Modules/ThirdParty/GoogleTest/src/itkgoogletest/googletest/cmake/internal_utils.cmake

```diff
@@ -271,55 +271,55 @@ endfunction()
+# NOT NEEDED FOR ITK           add_test(NAME ${name}
+# NOT NEEDED FOR ITK             COMMAND powershell -Command ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/RunTest.ps1
+# NOT NEEDED FOR ITK               ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
+# NOT NEEDED FOR ITK               --build_dir=${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG> ${ARGN})
+# NOT NEEDED FOR ITK         else()
+# NOT NEEDED FOR ITK           add_test(NAME ${name}
+# NOT NEEDED FOR ITK             COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
+# NOT NEEDED FOR ITK               --build_dir=${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG> ${ARGN})
+# NOT NEEDED FOR ITK         endif()
+# NOT NEEDED FOR ITK       else (CMAKE_CONFIGURATION_TYPES)
+# NOT NEEDED FOR ITK         # Single-configuration build generators like Makefile generators
+# NOT NEEDED FOR ITK         # don't have subdirs below CMAKE_CURRENT_BINARY_DIR.
+# NOT NEEDED FOR ITK         if (WIN32 OR MINGW)
+# NOT NEEDED FOR ITK           add_test(NAME ${name}
+# NOT NEEDED FOR ITK             COMMAND powershell -Command ${CMAKE_CURRENT_BINARY_DIR}/RunTest.ps1
+# NOT NEEDED FOR ITK               ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
+# NOT NEEDED FOR ITK               --build_dir=${CMAKE_CURRENT_BINARY_DIR} ${ARGN})
+# NOT NEEDED FOR ITK         else()
+# NOT NEEDED FOR ITK           add_test(NAME ${name}
+# NOT NEEDED FOR ITK             COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
+# NOT NEEDED FOR ITK               --build_dir=${CMAKE_CURRENT_BINARY_DIR} ${ARGN})
+# NOT NEEDED FOR ITK         endif()
+# NOT NEEDED FOR ITK       endif (CMAKE_CONFIGURATION_TYPES)
+# NOT NEEDED FOR ITK     else()
+# NOT NEEDED FOR ITK       # ${CMAKE_CURRENT_BINARY_DIR} is known at configuration time, so we can
+# NOT NEEDED FOR ITK       # directly bind it from cmake. ${CTEST_CONFIGURATION_TYPE} is known
+# NOT NEEDED FOR ITK       # only at ctest runtime (by calling ctest -c <Configuration>), so
+# NOT NEEDED FOR ITK       # we have to escape $ to delay variable substitution here.
+# NOT NEEDED FOR ITK       if (WIN32 OR MINGW)
+# NOT NEEDED FOR ITK         add_test(NAME ${name}
+# NOT NEEDED FOR ITK           COMMAND powershell -Command ${CMAKE_CURRENT_BINARY_DIR}/RunTest.ps1
+# NOT NEEDED FOR ITK             ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
+# NOT NEEDED FOR ITK             --build_dir=${CMAKE_CURRENT_BINARY_DIR}/\${CTEST_CONFIGURATION_TYPE} ${ARGN})
+# NOT NEEDED FOR ITK       else()
+# NOT NEEDED FOR ITK         add_test(NAME ${name}
+# NOT NEEDED FOR ITK           COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test/${name}.py
+# NOT NEEDED FOR ITK             --build_dir=${CMAKE_CURRENT_BINARY_DIR}/\${CTEST_CONFIGURATION_TYPE} ${ARGN})
+# NOT NEEDED FOR ITK       endif()
+# NOT NEEDED FOR ITK     endif()
+# NOT NEEDED FOR ITK   endif(PYTHONINTERP_FOUND)
+# NOT NEEDED FOR ITK endfunction()
```

Comment:
Is the above to be removed? or does it need an update for wrapping?

Reply:
I guess it would be easier to resolve conflicts in the future if unneeded code was removed. Perhaps replace this whole block with a single comment line stating that it is not needed, and perhaps a short explanation why - next time someone tries to update GTest they will be reading it.